src/app/services/template/mail/getFeedbackNotifyTemplate.ts

```diff
@@ -22,13 +22,20 @@ import * as path from 'path';
 import MailTemplate from 'app/template/MailTemplate';
 
 import * as app from 'app/util/applib';
+import Website from 'base/Website';
+import emptyCheck from 'app/util/emptyCheck';
 
 const log = app.createLog();
 const defaultTemplate = path.join('templates', 'mail', 'defaultFeedbackNotify.html');
 
-export default function() : Promise <MailTemplate> {
+export default function(website : Website) : Promise <MailTemplate> {
+	emptyCheck(website);
+
 	const rawTemplate = () : Promise <string> => {
```

Comment:
This time I have to agree with Codeclimate. The code in `getFeedbackNotifyTemplate()` is copied from `getFeedbackPageTemplate()` and at least the expression in the `return` statement can be refactored into a common function. Something like
```
return chooseTemplate(website.layout.templates.feedbackNotificationMail, defaultTemplate);
```

Reply:
Also `isEmpty` isn't imported from _lodash_ here, so the build doesn't pass. Will go away when the expression is refactored out into its own module.